README.md

```diff
@@ -0,0 +1,116 @@
+# Project TINA
+
+## Govhack 2024
+
+Project TINA is designed as an AI agent that sits between a constituent and any Local, State or Federal Government Department, Agency, or Service to provide a conversational interface that better facilitates the communication of constituent issues and the resolution of those issues.
+
+#### Authors
+
+[Daniel](dbryar@gmail.com) & [Louisa Bryar](louisa.bryar@gmail.com)
+
+## Introduction
+
+People _love_ to complain about government services. And not just the services, but the way their responsibilities are delivered within the context of the individual.
+
+> - _"Why can't the government fix all these potholes in our roads?"_
+> - _"How is it that the government can spend $5 billion on domestic violence prevention programs and I can still hear my neighbours at each other every night?"_
```

Comment:
very political. this is a policing matter and a matter of law not funding issues for the government.